### PR TITLE
Creation of an AUTHORS file.

### PR DESCRIPTION
**Pull request by UF-Buck-GitHub**
We need to make sure all projects have an AUTHORS file. I've created the standard file as used on other projects.

### CODE REVIEW

**Review by senrabc — CHANGES_REQUESTED** [review]
You need to update the text to reflect this is a honeyguide AUTHORS file

"Pigeon is used to copy redcap records into a new
+redcap project instance."

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Honeyguide Authors
+
+The RED-I Project would not be possible without the generous contributions of
+our authors and contributors. Pigeon is used to copy redcap records into a new
```
Comment (PFWhite) [inline AUTHORS]: I think we should call it honeyguide

**Review by PFWhite — CHANGES_REQUESTED** [review]
looks great!

File: AUTHORS (hunk 1 above)
Comment (UF-Buck-GitHub) [reply AUTHORS]: Looks good.

**Review by PFWhite — APPROVED** [review]

**Pull request merged**